pyproject.toml

```diff
@@ -0,0 +1,45 @@
+# With this the user can use a custom
+# python environment, like those created
+# with venv or pyenv. The deault will be
+# the system python.
+[virtualenvs]
+prefer-active-python = true
+
+# This isnt really needed on pratical aspect
```

Comment:
```diff
- # This isnt really needed on pratical aspect
+ # This isn't really needed on practical aspect
```

Reply:
changing the line below `s/organized/organize` to

Reply:
fixed